en/asgardeo/docs/apis/admin-advisory-management-rest-api.md

```diff
@@ -0,0 +1,5 @@
+---
+template: templates/redoc.html
+---
+
+<redoc spec-url="{{base_path}}/apis/restapis/admin-advisory-management.yaml" theme='{{redoc_theme}}'></redoc>
```

Comment:
This is not required for Asgardeo.
This API is only for super tenant

Reply:
https://github.com/wso2/docs-is/pull/4563/commits/eb753148eb4ef8762733cd878e6c2f1df7255557